### PR TITLE
Kreon: v2.000 added

### PR DESCRIPTION
**Pull request by eliheuer**
Taken from the upstream repo https://github.com/googlefonts/kreon 
at commit https://github.com/googlefonts/kreon/commit/467d2f024dcc338b941de2ed1e6e5eaafc189ceb

This is a new PR from `google/fonts:kreon-vf` that replaces an old PR from `eliheuer/fonts:kreon-vf`.
The old PR will be closed soon and can be found here: https://github.com/google/fonts/pull/1736

### CODE REVIEW

File: ofl/kreon/METADATA.pb [hunk 1]
```diff
@@ -1,34 +1,23 @@
 name: "Kreon"
 designer: "Julia Petretta"
 license: "OFL"
-category: "SERIF"
-date_added: "2011-02-02"
+category: "SANS_SERIF"
```
Comment (m4rc1e) [inline ofl/kreon/METADATA.pb]: Category should be SERIF and date should be the same.

Comment (eliheuer) [reply ofl/kreon/METADATA.pb]: Ok, this has been updated in commit: b100e09

**Comment by eliheuer** [comment]
This has been updated and is ready for review again. Thanks Marc.

**Comment by eliheuer** [comment]
I have updated this forthe new VF naming spec as was discussed here: https://github.com/google/fonts/issues/1817

**Pull request merged**